### PR TITLE
Upgrading packages and removing lib from source control

### PR DESCRIPTION
**Pull request by cereallarceny**
- Upgraded all dependencies to current versions.
- Test and build commands are still working perfectly.
- Also removed the `lib` folder from source control (it will still be pushed to NPM when a new version is released).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,38 +49,39 @@
   },
   "homepage": "https://github.com/yusinto/universal-hot-reload#readme",
   "dependencies": {
-    "@types/webpack": "^4.32.1",
-    "babel-loader": "^8.0.6",
+    "@types/webpack": "4.41.21",
+    "babel-loader": "8.1.0",
+    "clear-require": "^3.0.0",
```
Comment (yusinto) [inline package.json:54]: Does this need to omit the ^ to be consistent with the other package versions?

Comment (cereallarceny) [reply package.json:54]: No, this should be fine as we'll always want the latest version. I think it's fine.

**Comment by cereallarceny** [comment]
Merge when ready @yusinto 😄

**Comment by cereallarceny** [comment]
Is there anything else I need to do to get this merged @yusinto? I'm waiting on this and the other PR for my own deployment.

**Pull request merged**